overlord/devicestate/handlers_install.go

```diff
@@ -87,6 +88,28 @@ func writeModel(model *asserts.Model, where string) error {
 	return asserts.NewEncoder(f).Encode(model)
 }
 
+func writeLogs(rootdir string) error {
+	// XXX: would be great to use native journal but it's tied
+	//      to machine-id, we could journal -o export but there
+	//      is no systemd-journal-remote on core{,18,20}
+	//
+	// XXX: or only log if persistant journal is enabled?
+	logPath := filepath.Join(rootdir, "_writable_defaults/var/log/install-mode.log")
```

Comment:
this feels a bit like an abuse of the _writable_defaults directory since this file isn't really a writable "default" setting for anything, we are more using it here because it just so happens to be copied into /var/log when the initramfs runs so we can sneak files into the actual run mode system's writable /var.

Then again maybe it's fine because there's probably not a great place to store it otherwise.